packages/babel-traverse/test/modification.js

```diff
@@ -0,0 +1,33 @@
+import traverse from "../lib";
+import assert from "assert";
+import { parse } from "babylon";
+import * as t from "babel-types";
+import generate from "babel-generator";
+
+describe("path/modification", function () {
+  describe("insertBefore", function () {
+    const ast = parse(`function abc() {
+      var x = 3;
+    }`);
+
+    it("does things", function () {
+      traverse(ast, {
+        BlockStatement(path) {
+          path.insertBefore(t.expressionStatement(
+            t.callExpression(
+              t.identifier("s"),
+              [],
+            )
+          ));
+        },
+      });
+
+      assert.equal(generate(ast).code, `function abc() {
+  s();
+  {
```

Comment:
Oh I forgot about this PR because test coverage decreased while all I tried to do was adding tests. Looks like path still has relatively low coverage so I will look into increasing coverage here again. I think a new PR with more tests would make more sense so feel free to close this one.